app/models/role.rb

```diff
@@ -47,6 +47,14 @@ class Role < ActiveRecord::Base
 
   has_many :permissions, :through => :filters
 
+  # these associations are not used by Taxonomix but serve as a pattern for role filters
+  taxonomy_join_table = :taxable_taxonomies
```

Comment:
I added extra comment so now it's hopefully clear that we don't want to include Taxonomix into Role